ee/codegen/src/serializers/vellum.ts

```diff
@@ -1864,6 +1864,7 @@ export const GenericNodeSerializer: ObjectSchema<
   GenericNodeSerializer.Raw,
   Omit<GenericNode, "type">
 > = objectSchema({
+  id: stringSchema(),
```

Comment:
Generic node schema was missing its id. I plan to swap out all of the nodes in the graph attribute tests eventually with Generic nodes